src/frontend/web_application/src/components/Dropdown/presenter.jsx

```diff
@@ -11,10 +11,19 @@ Raw.propTypes = {
   children: PropTypes.node.isRequired,
 };
 
-const Presenter = ({ className, ...props }) => {
+
+const Presenter = ({ className, flat, ...props }) => {
+  const DropdownClassName = classnames(
+    'm-dropdown',
+    {
+      'm-dropdown--flat': flat,
```

Comment:
no flat modifier, use Raw instead (it is not well named or not enough documentation :(, my bad )